modules/runtime/pages/bonita-docker-installation.adoc

```diff
@@ -168,25 +168,23 @@ services:
       - |
         set -e
         echo 'Waiting for Postgres to be available'
-        export PGPASSWORD="$$POSTGRES_ENV_POSTGRES_PASSWORD"
+        export PGPASSWORD=$${POSTGRES_ENV_POSTGRES_PASSWORD}
         maxTries=10
-        while [ "$$maxTries" -gt 0 ] && ! psql -h "$$DB_HOST" -U 'postgres' -c '\l'; do
+        while [[ maxTries -gt 0 ]] && ! psql -h $${DB_HOST} -U 'postgres' -c '\l'; do
             let maxTries--
             sleep 1
         done
-        echo
-        if [ "$$maxTries" -le 0 ]; then
+        if [[ maxTries -le 0 ]]; then
```

Comment:
maxTries should be define as variable `$maxTries`, no ?

Reply:
it's define above (172)

Reply:
I reverted that part: both `"$$maxTries"` and `maxTries` work. Not `$maxTries`

Reply:
ok